gaj23/face-it
# Kby feature details component

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 
I created the Details component and the Single Product component in order to display the individual product details page. 


### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.
I added two folders and files - Details Component and Single Product Component. Inside of those folders you will find a JS file and a CSS file. The Details component is basically the "container" of the individual Single Product page. The Details component is what is routed inside of App.js


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?
App.js - review the Route path on line 425. When the route path matches the ID of the product, it will render the Details Component which is essentially the container rendering the SingleProduct component.
Details.js - rendering the SingleProduct page with details being passed down as props. Perform a find to find the specific product the page is on and pass down those properties to Single product to display.
SingleProduct.js - only rendering a div including the properties we want to display.


### How should this be tested?
> What steps need to be taken to confirm functionality? 
npm start => browser window => choose a category (Eco, Vegan, AlergenFriendly) => choose a product type (eyeshadow, bronzer, etc...) => choose a product => details page is shown

### Steps To Take
> Any questions and possible steps moving forward.



#### Checklist:
